uber/templates/registration/staffers.html

```diff
@@ -8,23 +8,25 @@
 </ol>
 
 {% if search_text %}
-        <div class="panel panel-info">
-    <a href="index?order={{ order }}">Click here</a> to view full volunteer list instead of only search results.
+    <div class="panel panel-info">
+        <a href="index?order={{ order }}">Click here</a> to view full volunteer list instead of only search results.
     </div>
 {% endif %}
 
-<div class="row">
-		<div class="col-md-6">
-<div class="input-group">
-<input class="form-control" type="text" name="filter" id="filter" placeholder="Type Here to Search"/>
-<input type="hidden" name="order" value="{{ order }}" />
-<span class="input-group-addon"><span class="glyphicon glyphicon-search"></span></span>
-</div>
-		</div>
-    <div class="col-md-2">{{ staffer_count }} volunteers</div>
-    <div class="col-md-2">{{ taken_hours }} shift hours taken</div>
-    <div class="col-md-2">{{ total_hours }} shift hours exist</div>
-</div>
+<form method="get" action="staffers">
+    <div class="row">
+        <div class="col-md-6">
+            <div class="input-group">
+                <input class="form-control" type="text" name="search_text" value="{{ search_text }}" placeholder="Type Here to Search"/>
+                <input type="hidden" name="order" value="{{ order }}" />
+                <span class="input-group-addon"><span class="glyphicon glyphicon-search"></span></span>
+            </div>
+        </div>
+        <div class="col-md-2">{{ staffer_count }} volunteers</div>
+        <div class="col-md-2">{{ taken_hours }} shift hours taken</div>
+        <div class="col-md-2">{{ total_hours }} shift hours exist</div>
+    </div>
+</form>
```

Comment:
Why does the form encapsulate so much? Wouldn't it be more sensible to put it just around the input-group div?